src/editors/containers/EditorContainer/components/CancelConfirmModal.tsx

```diff
@@ -9,6 +9,31 @@ interface CancelConfirmModalProps {
   onCloseEditor: (() => void) | null,
 }
 
+interface ActionButtonsProps {
+  closeCancelConfirmModal: () => void,
+  onCloseEditor: (() => void) | null,
+}
+
+const ActionButtons = ({
+  closeCancelConfirmModal,
+  onCloseEditor,
+}: ActionButtonsProps) => (
+  <div className="w-100 d-flex justify-content-between">
+    <Button
+      variant="outline-brand"
+      onClick={() => onCloseEditor?.()}
+    >
+      <FormattedMessage {...messages.discardChangesButtonlabel} />
+    </Button>
+    <Button
+      variant="primary"
+      onClick={closeCancelConfirmModal}
+    >
+      <FormattedMessage {...messages.keepEditingButtonLabel} />
+    </Button>
+  </div>
```

Comment:
```suggestion
  <>
    <Button
      variant="outline-brand"
      onClick={() => onCloseEditor?.()}
    >
      <FormattedMessage {...messages.discardChangesButtonlabel} />
    </Button>
    <ActionRow.Spacer />
    <Button
      variant="primary"
      onClick={closeCancelConfirmModal}
    >
      <FormattedMessage {...messages.keepEditingButtonLabel} />
    </Button>
  </>
```

If we apply the previous change, this can be simplified with just a spacer in between.

**Make sure to import `ActionRow`**

Reply:
Looks much better :+1: